orga/translations/en.json

```
@@ -1,15 +1,56 @@
 {
   "current-lang": "en",
+  "common": {
+    "form": {
+      "answer": {
+        "no": "No",
+        "yes": "Yes"
+      },
+      "action": {
+        "cancel": "Cancel"
+      }
+    }
+  },
   "navigation": {
-    "main" : {
-      "campaigns" : "Campaigns",
+    "main": {
+      "campaigns": "Campaigns",
       "documentation": "Documentation",
-      "students-sco" :"Students",
-      "students-sup" :"Students",
-      "team" : "Team"
+      "students-sco":"Students",
+      "students-sup":"Students",
+      "team": "Team"
     },
     "user-logged-menu": {
       "logout": "Log out"
     }
+  },
+  "pages": {
+    "create-campaign": {
+      "main-title": "Create a campaign",
```

Celui-ci pour le coup je pense que ça sera le même titre sur l'onglet
```suggestion
      "title": "Create a campaign",
```

---

Et du coup ne pas oublier de le changer dans l'ordre alphabétique ^^